Source/Models/Highlight+Helper.swift

```diff
@@ -289,15 +289,11 @@ extension Highlight {
     ///   - page: The page containing the HTML.
     ///   - highlightId: The ID to be removed
     /// - Returns: The removed id
-    @discardableResult public static func removeFromHTMLById(withinPage page: FolioReaderPage?, highlightId: String) -> String? {
-        guard let currentPage = page else { return nil }
+    public static func removeFromHTMLById(withinPage page: FolioReaderPage?, highlightId: String, completion: @escaping JSCallback) {
+        guard let currentPage = page else { return }
+        
+        currentPage.webView?.js("removeHighlightById('\(highlightId)')", completion: completion)
```

Comment:
Same here, if the completion is optional you can ommit it.
```suggestion
    public static func removeFromHTMLById(withinPage page: FolioReaderPage?, highlightId: String, completion: JSCallback? = nil) {
        guard let currentPage = page else { return }
        currentPage.webView?.js("removeHighlightById('\(highlightId)')", completion: completion)
```